### PR TITLE
Crl/intermediate expansion

### PR DESCRIPTION
**Pull request by ChloeL19**
Implemented intermediate expansion script and added to experiments.py

### CODE REVIEW

File: run_whole.py [hunk 1]
```diff
@@ -1,61 +0,0 @@
-from cmdline import args
```
Comment (namin) [inline run_whole.py:1]: why was the file deleted?

Comment (ChloeL19) [reply run_whole.py:1]: because I am trying to be neat with my PRs. It is included in my latest PR

Comment (namin) [reply run_whole.py:1]: But this is an existing file that we used for the benchmarks...

Comment (ChloeL19) [reply run_whole.py:1]: it is? it shouldn't be....

**Pull request merged**